exercises/pascals-triangle/instructions.md

```diff
@@ -0,0 +1,3 @@
+# Instructions
+
+Your task is to output the first N rows of Pascal's Triangle.
```

Comment:
```suggestion
Your task is to output the first N rows of Pascal's triangle.
```
Minor capitalization correction in line with the introduction and how Wikipedia cases it.